docs/CHANGELOG/changelog-0.12.x.md

```
@@ -0,0 +1,67 @@
+
+
+## v0.12.0
+
+## What's Changed
+
+- Changing credentials for providers will now be correctly propagated within the reconciliation loop [#2056](https://github.com/berops/claudie/pull/2056)
+
+- Updated MongoDB to version 6.0 [#2053](https://github.com/berops/claudie/pull/2053)
+
+    - After deploying, verify Mongo version is `6.0`
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)